internal/proxy/envoy/xds/ads/ads.go

```diff
@@ -0,0 +1,24 @@
+package ads
+
+import (
+	"context"
+	"time"
+
+	"github.com/goto/shield/core/rule"
+)
+
+const (
+	CLUSTER_TYPE_URL             = "type.googleapis.com/envoy.config.cluster.v3.Cluster"
```

Comment:
can we reuse the url from this repo instead? https://github.com/envoyproxy/go-control-plane/blob/fc612d4a3afabcbec1906fcd28df0f46adf6999b/pkg/resource/v3/resource.go#L19